FogCreek/Glitch-Community
# Magnetic-dish: a11y fixes & invisible text on homepage

## Links
* magnetic-dish.glitch.me
* [A11y issues](https://app.clubhouse.io/glitch/story/3154/minor-a11y-issues-on-homepage) and [invisible sentence](https://app.clubhouse.io/glitch/story/3391/invisible-sentence-on-the-homepage)

## GIF/Screenshots:
![image](https://user-images.githubusercontent.com/4480480/67801594-7a273780-fa57-11e9-8005-980dd6558c9a.png)

## Changes:
* adds aria-label to landmark regions, to help ensure landmarks are unique on each page.  Screenreader rotor menus, which are quite hard to screenshot, would otherwise possibly show 2 'main' elements, one from Glitch and one from the embed on the page. Now they'll show 'main Glitch' or 'main Glitch user page' alongside the 'main' coming from an embed that uses that landmark. 
* Fixes heading order on homepage
* fixes issue where the no-js fallback html creates duplicate `main` and `header` elements on the page. 
* fixes issue where the subtitle for the culture zine section on the homepage doesn't show up visually despite being in the html source

## How To Test:
* Run axe-core on the homepage, see a few less errors than when running on glitch.com
